pages/thankyou.js

```diff
@@ -45,22 +46,27 @@ export default function Confirmation(props) {
           </span>
           <div className="lg:w-2/3">
             <p className="mb-4 text-sm lg:text-p leading-30px">
-              {t("confirmationP1")} {maskedEmail} {t("from")}&nbsp;
+              {t("sentEmailTo")} {maskedEmail} {t("from")}{" "}
               <a
                 className="underline hover:text-canada-footer-hover-font-blue text-canada-footer-font"
                 href={`mailto: ${process.env.NEXT_PUBLIC_THANK_YOU_EMAIL}`}
               >
                 {process.env.NEXT_PUBLIC_THANK_YOU_EMAIL}
-              </a>
+              </a>{" "}
+              {t("toCheckEmail")}
             </p>
             <p className="mb-4 text-sm lg:text-p leading-30px">
-              {t("confirmationP2")}
+              {referrer === "signup"
+                ? t("confirmationP2")
+                : t("unsubscribeConfirmation")}
             </p>
             <p className="mb-4 text-sm lg:text-p leading-30px">
-              {t("confirmationP3")}
+              {referrer === "signup"
+                ? t("confirmationP3")
+                : t("unsubscribeConfirmationP1")}
             </p>
             <p className="mb-4 text-sm lg:text-p leading-30px">
-              {t("confirmationP4")}
+              {referrer === "signup" ? t("confirmationP4") : ""}
```

Comment:
If the referrer is not "signup" it will be an empty string. I think it would be better to place the `<p>` tag in the "true" section of the ternary operator.

Reply:
Good catch here!